### PR TITLE
[ONEM-19554] getList basic implementation

### PR DESCRIPTION
**Pull request by mariusz-sikora-red**
Changed sql queries into prepared statements
Changed getStorageDetails into kB

### CODE REVIEW

File: LISA/Debug.h [hunk 1]
```diff
@@ -54,7 +54,7 @@ void lisaInternalMakeLogMessage(Args&&... args)
 /** INTERNAL HELPERS END ***************************************************************/
 
 // use TRACE_L1 - for easier development, no need to set TRACE after every restart
-//#define FORCE_TRACE_L1_DEBUGS
+#define FORCE_TRACE_L1_DEBUGS
```
Comment (szymon-polke-red) [inline LISA/Debug.h:57]: pushed by accident?

Comment (mariusz-sikora-red) [reply LISA/Debug.h]: That's true. Fixed.

**Review by piotr-marcinkowski-red — APPROVED** [review]
Looks good, nice work :)

File: LISA/Executor.cpp [hunk 2]
```diff
@@ -198,12 +198,23 @@ uint32_t Executor::GetStorageDetails(const std::string& type,
...
 }
 
+uint32_t Executor::GetAppDetailsList(const std::string& type,
+                          const std::string& id,
+                          const std::string& version,
+                          const std::string& appName,
+                          const std::string& category,
+                          std::vector<DataStorage::AppDetails>& appsDetailsList) const
+{
+    appsDetailsList = dataBase->GetAppDetailsList(type, id, version, appName, category);
```
Comment (szymon-polke-red) [inline LISA/Executor.cpp:214]: add try {} catch for exceptions that might be thrown and return error, you can do the same for GetStorageDetails

Comment (mariusz-sikora-red) [reply LISA/Executor.cpp]: done

**Review by szymon-polke-red — APPROVED** [review]

**Pull request merged**